packages/varlet-ui/src/bottom-navigation-item/BottomNavigationItem.vue

```diff
@@ -53,7 +44,7 @@ export default defineComponent({
   props,
   setup(props) {
     const name = computed<string | undefined>(() => props.name)
-    const badge = computed<boolean | BadgeProps>(() => props.badge)
+    const isActive = computed<boolean>(() => active.value === name.value || active.value === index.value)
```

Comment:
Don't use conditional branch statements in computed. It is easy to create unnecessary mental burden due to dependence on collection issues.

Reply:
sorry, I forget it. It seems that using `includes` is a better choice.

Reply:
done